lib/plugins/aws/deploy/compile/events/apiGateway/README.md

```diff
@@ -52,3 +52,40 @@ functions:
                 path: posts/create
                 method: post
 ```
+
+### Http setup with custom authorizer
+You can enable custom authorizers for your HTTP endpoint with the following config:
+
+```yml
+# serverless.yaml
+functions:
+    create:
+        handler: posts.create
+        events:
+            - http:
+                path: posts/create
+                method: post
+                authorizer: authorizerFunc
+    authorizerFunc:
+        handler: handlers.authorizerFunc
+```
+Or, if you want to config the authorizer, you can do:
```

Comment:
please use configure instead of config and something a little more descriptive than "you can do", e.g. "If you want to configure your authoriser you can use the arguments shown in the following example:"